src/main/java/com/github/technus/tectech/recipe/EyeOfHarmonyRecipeStorage.java

```diff
@@ -0,0 +1,156 @@
+package com.github.technus.tectech.recipe;
+
+import static com.github.technus.tectech.recipe.EyeOfHarmonyRecipe.processHelper;
+import static com.github.technus.tectech.recipe.TT_recipe.GT_Recipe_MapTT.sEyeofHarmonyRecipes;
+import static java.lang.Math.pow;
+
+import com.github.technus.tectech.util.ItemStackLong;
+import com.google.common.math.LongMath;
+import gregtech.api.enums.Materials;
+import gregtech.api.enums.OrePrefixes;
+import gregtech.api.util.GT_OreDictUnificator;
+import java.util.*;
+import net.minecraft.block.Block;
+import net.minecraft.item.ItemStack;
+import org.apache.commons.lang3.tuple.Pair;
+import pers.gwyog.gtneioreplugin.plugin.block.BlockDimensionDisplay;
```

Comment:
This just feels so wrong... even if it works.

Reply:
Not sure what the problem is here?